### ISSUE TITLE
<Page> with upgrade prop throws warnings in console

### ISSUE THREAD

**Issue by jurasm** [wix/wix-style-react#3514]
# 🐛 Bug Report

### 🏗 Relevant Components

`<Page>`
...

### 😯 Current Behavior

When using `<Page upgrade>` React throws warning in console:
```
react.development.js:276 Warning: Each child in a list should have a unique "key" prop.

Check the render method of `Page`. See https://fb.me/react-warning-keys for more information.
    in div (created by Page)
    in Page (created by Page)
    in Page (created by Exam)
    in div (created by Exam)
    in Exam (created by Context.Consumer)
    in Route (created by App)
    in Router (created by BrowserRouter)
    in BrowserRouter (created by App)
    in App
``` 
...

### 🤔 Expected Behavior

It should not throw warnings.

...

### 👣 Steps to Reproduce

`<Page upgrade />`

  - **WSR version**: `6.16.2`
  - **Yoshi version**: `^3.0.0`

### 👀 Severity
- Low

**Comment by shlomitc** [comment]
Can you please provide a reproducible example code?

**Comment by nayish** [comment]
Anything new with this issue? this really bloats our build log on CI and bothers when testing (too many warnings makes it hard to find the errors)
